### ISSUE TITLE
[FEATURE REQUEST] -Fader Sliders

### ISSUE THREAD

**Issue by Streethawk9081** [rivafarabi/deckboard#620]
Can you add slider / Fader type buttons in addition to the on/off type buttons.

These could be linked to faders on other software like VoiceMeeter or Elgato Wave or even the GoXLR.
The GoXLR does have 4 faders, but there are more faders in the background that would also be great to control externally.

**Comment by PifLyon** [comment]
Yes ! Yes ! Yes ! This would be a KILLER feature with voicemeeter !

It would definitivly burry Elgato's Streamdeck that will never be able to do that. :)

**Comment by Teraldar** [comment]
Yes Yes Yes! I want it so much! GoXLR and other hardware devices would be unnecessary then!

**Comment by tinab0** [comment]
i saw an slider on a deckboard prind once, but i cant find how i get that
#687
the print is in this link, i really want this hahaha
